analyzer/tests/unit/test_option_parser.py

```diff
@@ -378,6 +379,12 @@ def test_ccache_compiler(self):
                          '/usr/lib/ccache/g++')
 
     def test_compiler_gcc_implicit_includes(self):
+
+        # OSX does not have 'include-fixed' in the list of implicit includes.
```

Comment:
It would be better to use the `@unittest.skipIf` decorator:
https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures. This way we would see in the test output that this test case was skipped.